### ISSUE TITLE
Setting a datapoint to the same value causes update to cloud

### ISSUE THREAD

**Issue by Xyolyp** [ioBroker/ioBroker.iot#527]
This german forum thread is related: https://forum.iobroker.net/topic/61546/1001-befehle-verwendet-wie-ursachen-finden

I observed the counter for commands today at https://iobroker.pro/www/account/diagnostics and did some experiments.
During these I observed the following behavior:
1. Have a datapoint for a roller shutter position and one for the actual reported back position
2. Set up a device through the devices adapter for the roller shutter and use the two datapoints for SET and ACTUAL
3. Expose that device through the IoT Adapter to Google Home
4. Set the SET datapoint to any value. For the sake of this example I'll use 100%
5. Wait for the roller shutter to move to 100% and confirm that move by updating the ACTUAL datapoint. Now SET and ACTUAL both have the value 100%
6. Note the "commands today" counter from the cloud diagnostics.
7. Write the same value of 100% to the SET datapoint. The physical roller shutter won't move but comparing the "commands today" counter with the previously noted down value it will have increased by one or sometimes even two.
No other Device was configured in the IoT adapter. 

I would expect the IoT adapter to only send data to the IoT Cloud when something changes (or the adapter starts and maybe another edge case I currently can't come up with). In my opinion this should at least be a configurable option in maybe one of the following three flavors:
A. A single checkbox in the adapter configuration.
B. A setting per exposed device
C. Both A and B combined where when nothing is specifically set in the exposed devices settings the adapters setting is used and if it is set for a specific device this setting is used.

I would favor C but understand that this is the greatest development and testing efforts. Between the other two I prefer A as I hope this would result in the lowest development and testing efforts and on top it is easy to set up as a user.

**Comment by Apollon77** [comment]
@GermanBluefox I would agree ...  "changed values" should only be reported to Google when they really change

**Comment by Xyolyp** [comment]
It's not only reporting to Google but even not reporting to the ioBroker cloud system what this issue is about. I guess this would lower operational costs as well if fewer messages have to be processed.

**Comment by Apollon77** [comment]
yes, but only for Google users (and these are not that many) ;-)) But yes we will look into this
